nexus/tests/integration_tests/projects.rs

```diff
@@ -72,3 +85,366 @@ async fn test_projects(cptestctx: &ControlPlaneTestContext) {
     assert_eq!(projects.len(), 1);
     assert_eq!(projects[0].identity.name, p1_name);
 }
+
+async fn delete_project_default_subnet(url: &str, client: &ClientTestContext) {
+    NexusRequest::object_delete(
+        &client,
+        &format!("{url}/vpcs/default/subnets/default"),
+    )
+    .authn_as(AuthnMode::PrivilegedUser)
+    .execute()
+    .await
+    .expect("failed to make request");
+}
+
+async fn delete_project_default_vpc(url: &str, client: &ClientTestContext) {
+    NexusRequest::object_delete(&client, &format!("{url}/vpcs/default"))
+        .authn_as(AuthnMode::PrivilegedUser)
+        .execute()
+        .await
+        .expect("failed to make request");
+}
+
+async fn delete_project(url: &str, client: &ClientTestContext) {
+    NexusRequest::object_delete(&client, url)
+        .authn_as(AuthnMode::PrivilegedUser)
+        .execute()
+        .await
+        .expect("failed to make request");
+}
+
+async fn delete_project_expect_fail(
+    url: &str,
+    client: &ClientTestContext,
+) -> String {
+    let error: HttpErrorResponseBody = NexusRequest::expect_failure(
+        client,
+        StatusCode::BAD_REQUEST,
+        Method::DELETE,
+        &url,
+    )
+    .authn_as(AuthnMode::PrivilegedUser)
+    .execute()
+    .await
+    .unwrap()
+    .parsed_body()
+    .unwrap();
```

Comment:
In the past I've found it a bit hard to figure out test failures with multiple `unwrap`s in a single statement. Could I convince you to replace these with `expect` calls? :)